content/en/_index.md

```diff
@@ -104,19 +121,37 @@ to interact with a returned object programmatically by printing some example
 attributes to the console.
 
 {{< tabpane langEqualsHeader=true >}}
- {{< tab header="Java" >}}
+  {{< tab header="Java" >}}
     public static void main(String[] args) {
         // TODO(developer): Replace these variables before running the sample.
         String projectId = "my-project-id";
         String filePath = "path/to/image.png";
-        inspectImageFile(projectId, filePath);
+        exampleSnippet(projectId, filePath);
     }
 
     // This is an example snippet for showing best practices.
     public static void exampleSnippet(String projectId, String filePath) {
         // Snippet content ...
     }
-{{< /tab >}}
+  {{< /tab >}}
+  {{< tab header="Node.js" >}}
+    // Required values are passed as arguments to the main() function.
+    // Example declarations are included (but commented out) in the snippet.
+    function main(projectId, filePath) {
```

Comment:
> Any declared function arguments should include a no-arg, main method

This shouldn't have arguments. However, I don't particularly care if it's called "main" since node doesn't require that - if it's more idiomatic to just call it "runSample" that's fine with me.

Reply:
I'm fine with `main`. Per our discussion in a different comment - replaced this with a no-arg `main` function and a call to `exports`.